drivers/st25r3911b/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
```

Comment:
Shouldn't it be "Nordic Semiconductor ASA" as in other copyright headers?

Reply:
I think so